<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculatorWidget</class>
 <widget class="QWidget" name="CalculatorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" rowspan="4">
    <widget class="CLaboratoryCalculatorTableView" name="tableView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lblMaxGroupValue">
     <property name="text">
      <string>Групповой предел: </string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="CMaxGroupValueSpinBox" name="edtMaxGroupValue">
     <property name="maximum">
      <number>9999999</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QWidget" name="pnlButtons" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="1" column="3" rowspan="2">
       <widget class="CControlButton" name="btnPlus">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>+</string>
        </property>
        <property name="shortcut">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="CControlButton" name="btnSix">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>6</string>
        </property>
        <property name="shortcut">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="CControlButton" name="btnEight">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>8</string>
        </property>
        <property name="shortcut">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="CControlButton" name="btnSeven">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>7</string>
        </property>
        <property name="shortcut">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="CControlButton" name="btnThree">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>3</string>
        </property>
        <property name="shortcut">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="CControlButton" name="btnTwo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>2</string>
        </property>
        <property name="shortcut">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="CControlButton" name="btnNine">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>9</string>
        </property>
        <property name="shortcut">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="CControlButton" name="btnFour">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>4</string>
        </property>
        <property name="shortcut">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="CControlButton" name="btnOne">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>1</string>
        </property>
        <property name="shortcut">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="CControlButton" name="btnFive">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>5</string>
        </property>
        <property name="shortcut">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="CControlButton" name="btnZero">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="shortcut">
         <string>0</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
        <property name="autoRaise">
         <bool>false</bool>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QToolButton" name="btnReset">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>.</string>
        </property>
        <property name="shortcut">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="CControlButton" name="btnDiv">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>/</string>
        </property>
        <property name="shortcut">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="CControlButton" name="btnMultiply">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>*</string>
        </property>
        <property name="shortcut">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="CControlButton" name="btnMinus">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>-</string>
        </property>
        <property name="shortcut">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3" rowspan="2">
       <widget class="QToolButton" name="btnEnter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>E</string>
        </property>
        <property name="shortcut">
         <string>Enter</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="4">
       <widget class="QToolButton" name="btnCancel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Отменить</string>
        </property>
        <property name="shortcut">
         <string>Backspace</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="4">
       <widget class="QToolButton" name="btnResetAdditional">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Отменить назначения</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="lblRounding">
     <property name="text">
      <string>Округление</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QSpinBox" name="edtRounding">
     <property name="maximum">
      <number>9</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CLaboratoryCalculatorTableView</class>
   <extends>QTableView</extends>
   <header>TissueJournal/LaboratoryCalculator/LaboratoryCalculatorTable.h</header>
  </customwidget>
  <customwidget>
   <class>CMaxGroupValueSpinBox</class>
   <extends>QSpinBox</extends>
   <header>TissueJournal/LaboratoryCalculator/Utils.h</header>
  </customwidget>
  <customwidget>
   <class>CControlButton</class>
   <extends>QToolButton</extends>
   <header>TissueJournal/LaboratoryCalculator/Utils.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tableView</tabstop>
  <tabstop>btnDiv</tabstop>
  <tabstop>btnMultiply</tabstop>
  <tabstop>btnMinus</tabstop>
  <tabstop>btnSeven</tabstop>
  <tabstop>btnEight</tabstop>
  <tabstop>btnNine</tabstop>
  <tabstop>btnPlus</tabstop>
  <tabstop>btnFour</tabstop>
  <tabstop>btnFive</tabstop>
  <tabstop>btnSix</tabstop>
  <tabstop>btnOne</tabstop>
  <tabstop>btnTwo</tabstop>
  <tabstop>btnThree</tabstop>
  <tabstop>btnEnter</tabstop>
  <tabstop>btnZero</tabstop>
  <tabstop>btnReset</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>edtMaxGroupValue</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
